libgnucash/engine/cap-gains.cpp

```diff
@@ -126,34 +126,34 @@ latest_pred (time64 earl, time64 tran)
 static gpointer
 finder_helper (GNCLot *lot,  gpointer user_data)
 {
-    struct find_lot_s *els = user_data;
+    struct find_lot_s *els = (struct find_lot_s *)user_data;
```

Comment:
Remove `struct` from both sides. In C++ `struct` defines a type just like `class`, so you can refer to it as just `find_lot_s`. Better still, also rename it to `FindLot` to comply with our naming policy.

Also prefer `static_cast<FindLot*>` to C-style `(FindLot*)`. If `static_cast` won't compile use `reinterpret_cast`.